### ISSUE TITLE
Drift_dev analyzer needs to be updated

### ISSUE THREAD

**Issue by frankvollebregt** [simolus3/moor#2419]
**Describe the bug**
When I try to run the `build_runner`, the logs as seen below are in the output, and it fails to run. 
```
Error: The non-abstract class '_AddFromDartType' is missing implementations for these members:
 - TypeVisitor.visitInvalidType
```
When looking at the repository of the analyzer, I can see that the change was made just a few days ago, in this commit:
https://github.com/dart-lang/sdk/commit/2e6bdc29de69eda869522bce094f6fe76f13bdf1

Meaning that the `_AddFromDartType` class will need to be updated with this new override (which, from what I understand, is similar to what `DynamicType` previously did).

```
> flutter pub run build_runner build
[INFO] Generating build script...
[INFO] Generating build script completed, took 323ms

[INFO] Precompiling build script......
[WARNING] ../../../AppData/Local/Pub/Cache/hosted/pub.dev/drift_dev-2.7.0/lib/src/analysis/results/dart.dart:249:7: Error: The non-abstract class '_AddFromDartType' is missing implementations for these members:
 - TypeVisitor.visitInvalidType
Try to either
 - provide an implementation,
 - inherit an implementation from a superclass or mixin,
 - mark the class as abstract, or
 - provide a 'noSuchMethod' implementation.

class _AddFromDartType extends TypeVisitor<void> {
      ^^^^^^^^^^^^^^^^
../../../AppData/Local/Pub/Cache/hosted/pub.dev/analyzer-5.12.0/lib/dart/element/type_visitor.dart:19:5: Context: 'TypeVisitor.visitInvalidType' is defined here
.
  R visitInvalidType(InvalidType type);
    ^^^^^^^^^^^^^^^^
[INFO] Precompiling build script... completed, took 800ms

[SEVERE] Failed to precompile build script .dart_tool/build/entrypoint/build.dart.
This is likely caused by a misconfigured builder definition.

pub finished with exit code 78
```

**Comment by frankvollebregt** [comment]
I just realised this change was already proposed as a PR by the author of the commit, so I'm closing this issue :)
https://github.com/simolus3/drift/pull/2417

**Comment by simolus3** [comment]
Welp, and the documentation explicitly states that we shouldn't extend that class... Oops.

**Comment by simolus3** [comment]
Fixed in `drift_dev: 2.8.0`.
